apps/readest-app/src-tauri/capabilities/default.json

```diff
@@ -23,6 +23,11 @@
       ],
       "deny": []
     },
+    {
+      "identifier": "fs:allow-read",
```

Comment:
This is not preferred and might raise security issues. We can allow specific file like this: https://github.com/readest/readest/blob/main/apps/readest-app/src-tauri/src/lib.rs#L29